### PR TITLE
Mentor can filter questions by tag using filter buttons

### PR DESCRIPTION
**Pull request by cockochan**
Thank you for contributing! To help the review process, please provide the following:

### Proposal
Mentor can filter questions by tag using filter buttons

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -244,7 +214,6 @@ td {
   cursor: pointer;
   padding: 5px;
 }
-
 .centered {
   justify-items: center;
   text-align: center;
```
Comment (naderakhgari) [inline client/src/App.css:219]: I think in css one line space after every class is helps to read the code 
I don't know why you delete the line spaces here ?

**Review by naderakhgari — CHANGES_REQUESTED** [review]
looks good 
but didn't understand why removed the line spaces in css ?

File: client/src/App.css (hunk 1 above)
Comment (cockochan) [reply client/src/App.css:219]: cause youasked todelete some spaces in css;)nthats the only reason

Comment (naderakhgari) [reply client/src/App.css:219]: good job Constantin, no worry, if we have a line space after a specific piece of code it helps to be more readable

**Pull request merged**